cdp_scrapers/instances/portland.py

```diff
@@ -116,6 +117,15 @@ def get_person(self, name: str) -> Optional[Person]:
         #       Should add to known_persons so we don't query this same person again.
         return None
 
+    def separate_name_from_title(self, title_and_name: str):
+        # title_and_name:
+        #   The title (Mayor of Commissioner) and name of a Portland City Commission
+        #   member.
+        #   e.g., Mayor Ted Wheeler, Commissioner Carmen Rubio
+
+        name_index = title_and_name.index(" ")
```

Comment:
I prefer `find()` since that returns -1 if not found, but `index()` raises an exception.